### ISSUE TITLE
Token expiry bug: users getting unauthenticated after a short period

### ISSUE THREAD

**Issue by tnt-freskim-veliu** [sidebase/nuxt-auth#417]
### Environment

------------------------------
- Operating System: Darwin
- Node Version:     v19.7.0
- Nuxt Version:     3.5.2
- Nitro Version:    2.3.3
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      env, head, css, ssr, target, postcss, plugins, modules, auth, runtimeConfig, imports
- Runtime Modules:  @pinia/nuxt@0.4.9, @sidebase/nuxt-auth@0.6.0-beta.2
- Build Modules:    -
------------------------------

### Describe the bug

For the backend part, I'm using Laravel v9.19, and for generating tokens, I'm using Laravel Sanctum v3.0

Despite setting the token lifetime in Laravel Sanctum to be 1 month, in the sanctum expiration config, users are unexpectedly logged out after a short period, usually around 2 hours. This behavior is causing inconvenience to users as they are being logged out prematurely, interrupting their sessions and necessitating frequent re-authentication.

### Additional context

This is my nuxt.config.js for auth:

```
   auth: {
        provider: {
            type: 'local',
            endpoints: {
                signIn: {path: '/login', method: 'post'},
                signUp: {path: '/register', method: 'post'},
                signOut: {path: '/logout', method: 'post'},
                getSession: {path: "/profile", method: "get"},
            },

            token: {
                signInResponseTokenPointer: '/data/token',
                type: 'Bearer',
                headerName: 'Authorization'
            },

            maxAgeInSeconds: 525600,
        },

        pages: {
            login: '/login'
        },
        baseURL: process.env.API_BASE_URL,

        session: {
            enableRefreshOnWindowFocus: false
        },
        maxAgeInSeconds: 525600
    },

```

**Comment by SilvioDoMine** [comment]
Have you found a solution for it?

**Comment by tnt-freskim-veliu** [comment]
> Have you found a solution for it?

Yes, it was my mistake adding: ```enableRefreshOnWindowFocus``` in the wrong json place
